### ISSUE TITLE
Debugging a non-Borg Bug

### ISSUE THREAD

**Issue by BloodBlight** [borgbackup/borg#7585]
## Have you checked borgbackup docs, FAQ, and open GitHub issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Question

## System information. For client/server mode post info for both machines.
Borg client on Pop!_OS 22.04 LTS  and a Ceph Cluster with CephFS running on Ubuntu 20.04 LTS

#### Your borg version (borg -V).
borg-linux64 1.2.3

#### Operating system (distribution) and version.
Pop!_OS 22.04 LTS 
and
Ubuntu 20.04 LTS

#### Hardware / network configuration, and filesystems used.
ext4 & CephFS

#### How much data is handled by borg?
N/A

#### Full borg commandline that lead to the problem (leave away excludes and passwords)
borg list /path/to/repo

## Describe the problem you're observing.
To start, this is almost certainly NOT a borg bug, but it triggers it consistently, so I am hoping borg has some super secret super verbose mode.

The TLDR of situation, I upgraded from Mint 20.03 to Pop OS for bleeding edge kernel support of my new GPU (probably the real issue), and now when I access a borg repo on a remotely mounted CephFS volume it locks up the mount point.

Using borg targeting a CephFS mount point on another system over SSH works perfectly.

Borg can see, and clear it’s past locks, but then just stops once the mount locks up. 
 
No other software seems to trigger the issue (that I have found), so it’s some sort of call borg is making that triggers it.  Any help in producing an extremely detailed debug log or trace would be greatly appreciated!

**Comment by ThomasWaldmann** [comment]
Yeah, this is likely a case of a malfunctioning filesystem.

You maybe can get more information by using `strace`.

**Comment by ThomasWaldmann** [comment]
Did you find out anything?
